packages/components/app/styles/components/tabs.scss

```diff
@@ -112,3 +112,8 @@
     transition-property: left, width;
   }
 }
+
+// Prevent consumers from accidentally overriding default styling of HTML “hidden” attribute
+.hds-tabs__panel[hidden] {
+  display: none !important;
+}
```

Comment:
would we be able to solve this specific case by increasing specificity, either by chaining with parent `.hds-tabs.hds-tabs__panel[hidden]` or chaining with itself `.hds-tabs__panel.hds-tabs__panel`?

Reply:
I deleted the !important. The person having the issue wasn't able to track down the exact cause but we can try this and if it doesn't fix the issue then try chaining together selectors.

Reply:
The Slack thread link has details of what I discussed with them: https://hashicorp.slack.com/archives/C7KTUHNUS/p1675731491282229

She wasn't able to track down the exact code causing the issue but she provided what she used to fix the issue on her side.